plugins/blockvolume/blockprovider/blockvolume_provider.go

```diff
@@ -46,7 +46,7 @@ func RegisterBlockProvider(name string, f ProviderFunc) {
 		log.WithField("name", name).Error("failed to register block provider, provider already exist")
 		return
 	}
-	log.WithField("name", name).Infof("Registered block provider")
+	log.WithField("name", name).Debug("Registered block provider")
```

Comment:
Does this mean that when we execute 'glusterd2 --version' all the logs will be captured in the console and we're fixing it by just changing the log level to debug. IOW, if we don't redirect the logs to the console, https://github.com/gluster/glusterd2/issues/1475 isn't a problem at all?

Reply:
logs are getting initialized in main func, and all block providers got initialized in init(). For this, we need to init logging before main getting start to execute. @aravindavk @Madhu-1 can you please suggest, should we init conf and logging before main gets start to execute?